public/js/main.js

```diff
@@ -40,6 +41,7 @@ const actions = bindActionCreators(require("./actions"), store.dispatch);
 // global for debugging purposes only!
 window.store = store;
 window.injectDebuggee = require("./test/utils/debuggee");
+window.serializeStore = () => JSON.parse(JSON.stringify(store.getState()));
```

Comment:
this is a nice to have so that you can easily inspect state... takes advantage of the serializability of immutable js